src/utils.ts

```diff
@@ -24,13 +24,8 @@ import {
     computeLedgerHash,
     computeEscrowHash,
     computePaymentChannelHash,
-} from '../common/hashes'
-import { generateAddressAPI, GenerateAddressOptions, GeneratedAddress } from '../offline/generate-address'
-
-// @deprecated Use X-addresses instead
-const generateAddress = (options: GenerateAddressOptions = {}): GeneratedAddress => (
-    generateAddressAPI({...options, includeClassicAddress: true})
```

Comment:
`generateAddressAPI` is a terrible name, we should change it

Reply:
Changed the name to generateXAddress since that's the default behavior